.circleci/config.yml

```diff
@@ -99,6 +100,145 @@ jobs:
       - store_artifacts:
           path: /tmp/screenshots
 
+  e2e-test-android:
```

Comment:
Lots of commands here are not on the [Android Detox Guide](https://github.com/wix/Detox/blob/master/docs/Introduction.Android.md).
If you got them elsewhere, make sure everything is really needed on our case and comment wherever possible.
It's ok to learn from other repos, but not ok to have unnecessary workarounds on our own.

Reply:
step by step I removed some commands and then run job. Now only required commands are left